I/ITK/build_tarballs.jl

```diff
@@ -10,13 +10,39 @@ script = raw"""
 if [[ "${target}" == *x86_64-w64-mingw32* ]]; then
     CONFIG=msys2-64
     OS=Windows
-    # Add Windows-specific flags
-    export CXXFLAGS="-DITK_LEGACY_REMOVE=OFF -DVNL_DLL_DATA= -DITK_EXPORTS ${CXXFLAGS}"
-    export CFLAGS="${CFLAGS} -DVNL_DLL_DATA= -DITK_EXPORTS"
-else
-    export CXXFLAGS="-DITK_LEGACY_REMOVE=OFF ${CXXFLAGS}"
-fi
+export LDFLAGS="-L${libdir}"
+cd $WORKSPACE/srcdir/ITK*
+mkdir build/
+cmake -B build -S . \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DITK_USE_SYSTEM_EXPAT:BOOL=ON \
+    -DITK_USE_SYSTEM_FFTW:BOOL=ON \
+    -DITK_USE_SYSTEM_HDF5:BOOL=ON \
+    -DITK_USE_SYSTEM_JPEG:BOOL=ON \
+    -DITK_USE_SYSTEM_TIFF:BOOL=ON \
+    -DITK_USE_SYSTEM_PNG:BOOL=ON \
+    -DITK_USE_SYSTEM_EIGEN:BOOL=ON \
+    -DITK_USE_SYSTEM_ZLIB:BOOL=ON \
+    -DQNANHIBIT_VALUE:BOOL=0 \
+    -DQNANHIBIT_VALUE__TRYRUN_OUTPUT:STRING=0 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
+    -DVCL_HAS_LFS:STRING=1 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license ${WORKSPACE}/srcdir/ITK/LICENSE
```

Comment:
Avoid duplication, you're running pretty much the same commands in the other branch too.